src/collection/resources/users.js

```diff
@@ -0,0 +1,57 @@
+const Components = require('./_resources');
+const User = require('../objects/user');
+const {
+  getFromId,
+  getAll,
+  get,
+  search,
+  set,
+  setOne,
+} = require('./_utils')(User);
+
+class Users extends Components {
+  constructor(request) {
+    super([getFromId, getAll, get, search, set, setOne]);
+    this.request = request;
+    this.apiPath = 'users';
+    this.ResConstructor = User;
+    this.setMe = () => new User(request);
+  }
+
+  create({ body, urlParams, json = false } = {}, done) {
```

Comment:
document or comment that they should just use the .authenticate, no need to use .create and then .authenticate